### ISSUE TITLE
Difft 0.50.0 does not run 

### ISSUE THREAD

**Issue by tmysik** [Wilfred/difftastic#556]
Thanks for reporting a bug! Please include all of the following:

(1) A description of the issue. A screenshot is often helpful too.
```
~ difft --version
difft: /lib64/libc.so.6: version `GLIBC_2.29' not found (required by difft)
```

(3) The version of difftastic you're using (see `difft --version`) and
your operating system.

Oracle Linux 8.
```
~ uname -r
4.18.0-477.21.1.el8_8.x86_64
```

Thanks!

**Comment by Wilfred** [comment]
The version of glibc on your distro is too old for the prebuilt packages I'm afraid. Try `cargo install`.

You might also be able to use the Fedora 8 packages, as I think Oracle Linux 8 is close to that.

**Comment by Wilfred** [comment]
I think this would be solved by #357, so closing in favour of that.

**Comment by tmysik** [comment]
MUSL build would be great, thanks!
